modules/observe/proc-admin-enabling-metrics-ocp-operator.adoc

```diff
@@ -25,24 +25,26 @@ Currently, the {product} Operator does not support creating a `ServiceMonitor` c
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
-  name: <custom_resource_name> # <1>
-  namespace: {my-product-namespace} # <2>
+  name: <service-monitor-name> <1>
+  namespace: {my-product-namespace} <2>
   labels:
-    app.kubernetes.io/instance: <custom_resource_name>
-    app.kubernetes.io/name: backstage
+    app.kubernetes.io/instance: {my-product-cr-name} <3>
+    app.kubernetes.io/name: {product-custom-resource-type}
 spec:
   namespaceSelector:
     matchNames:
-      - {my-product-namespace}
+      - {my-product-namespace} <2>
   selector:
     matchLabels:
-      rhdh.redhat.com/app: backstage-<custom_resource_name>
+      app.kubernetes.io/instance: {my-product-cr-name} <3>
+      app.kubernetes.io/name: {product-custom-resource-type}
   endpoints:
   - port: http-metrics
     path: '/metrics'
 ----
-<1> Replace `<custom_resource_name>` with the name of your {product} CR.
-<2> Replace `<project_name>` with the name of the {ocp-short} project where your {product} instance is running.
+<1> The name of your ServiceMonitor resource, for example, `developer-hub-service-monitor`.
+<2> The namespace where your ServiceMonitor lives and where your {product-very-short} instance is installed.
+<3> The name of the {ocp-short} project where your {product} instance is running.
```

Comment:
This description is a bit confusing, it is the name of the RHDH deployment (something like `developer-hub`) - I'm not sure what the standard value is used in the docs.